src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,7 +6,7 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public abstract class Address implements Printable{
```

Comment:
Should this class be abstract?

Reply:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#layout) violation!